app/assets/stylesheets/components/scratchpad.css.scss

```
@@ -59,31 +59,26 @@
 }
 
 // Wrapper and its child should take up as much space as possible
-#scratchpad-editor-wrapper,
-#scratchpad-editor-wrapper>div:first-child {
-    height: 100%;
-    max-height: 100%;
-}
-
-.cm-editor {
-    // Inform editor of height preferences
-    height: 100%;
-}
-
-// More specific selector to override !important property from Papyros
-div .cm-gutters {
-    // Height 100% seems to not work on the gutters, so override the default
-    height: auto !important;
-    min-height: 100% !important;
+#scratchpad-editor-wrapper , #scratchpad-editor-wrapper > div:first-child {
```

```suggestion
#scratchpad-editor-wrapper,
#scratchpad-editor-wrapper>div:first-child {
```
Style was made uniform in #3694, please follow it.